src/custom/pages/Claim/InvestmentFlow/index.tsx

```diff
@@ -118,19 +135,42 @@ export default function InvestmentFlow({ hasClaims, isAirdropOnly, ...tokenAppro
 
   return (
     <InvestFlow>
-      <StepIndicator>
-        <Steps step={investFlowStep}>
-          <li>Allowances: Approve all tokens to be used for investment.</li>
-          <li>Submit and confirm the transaction to claim vCOW</li>
-        </Steps>
-        <h1>
-          {investFlowStep === 0
-            ? 'Claiming vCOW is a two step process'
-            : investFlowStep === 1
-            ? 'Set allowance to Buy vCOW'
-            : 'Confirm transaction to claim all vCOW'}
-        </h1>
-      </StepIndicator>
+      <Stepper steps={STEPS_DATA} activeStep={investFlowStep} />
+
+      <h1>
+        {investFlowStep === 0
+          ? 'Claim and invest'
+          : investFlowStep === 1
+          ? 'Set allowance to Buy vCOW'
+          : 'Confirm transaction to claim all vCOW'}
+      </h1>
+
+      {investFlowStep === 0 && (
+        <p>
+          You have chosen to exercise one or more investment opportunities alongside claiming your airdrop. Exercising
+          your investment options will give you the chance to acquire vCOW tokens at at fixed price. This process
+          consists of two steps.
+          <br />
+          <br />
+          The first step allows you to define the investment amounts and set the required allowances for the tokens you
+          will use to invest with.
+          <br />
+          <br />
+          The last step executes all claiming opportunities on-chain. Additionally, it sends the tokens you will use to
+          invest with, to the smart contract. In return, the smart contract will send the vCOW tokens for the Airdrop
+          and your 4 years linear vesting of the investment amount will start.
+          <br />
+          <br />
+          For more details around the token, please read{' '}
+          <a href="https://cow.fi" target="_blank" rel="noreferrer">
+            the blog post
+          </a>
+          <br />. For more details about the claiming process, please read the{' '}
```

Comment:
As Elena noted, the period is after the line break

```suggestion
          </a>.
          <br /> For more details about the claiming process, please read the{' '}
```

Reply:
Nice find :)